src/dragSessionTracker.ts

```diff
@@ -149,27 +152,22 @@ export const dragSessionTracker = dragSessionModel.createMachine(
           return ctx.session;
         },
       }),
-      clearSessionData: assign({
+      clearSessionData: assign((ctx) => ({
```

Comment:
hm, this might be worth investigating - ideally this unused argument/function shouldn't be needed

Reply:
Btw. the event here is typed as `never` but that is incorrect - this action is an exit action used in the active state and this should be able to receive at least `DRAG_SESSION_STOPPED`